src/components/categories.js

```diff
@@ -0,0 +1,5 @@
+import React from 'react';
+
+const Categories = () => <div> Categories</div>;
```

Comment:
- [ ] Based on the requirements for this milestone, please ensure that your categories component display only a button with text "Check status"